build/webpack.client.base.conf.js

```diff
@@ -35,7 +35,11 @@ module.exports = merge.smart(baseWebpackConfig, {
 				'style': 'display:none;',
 			},
 			svgoOptions: {
-				plugins: [],
+				plugins: [
+					{
+						convertPathData: false,
```

Comment:
@emuvente We found that without this the svgs were being over-optimized to the point of losing and changing portions of the original image. Found this tip (https://github.com/svg/svgo/issues/1074) to fix that along with the options we may want to use in the future (https://github.com/svg/svgo/blob/master/examples/test.js)

Reply:
Which svgs did it affect? How much did the final icon bundle size increase? Another thing that may be responsible for the problems is the floatPrecision below. It might be worth experimenting with changing that to 3 (which is the svgo default), which might result in more favorable final file sizes without affecting the quality.

Reply:
We can go back and see how the file sizes were affected. Unfortunately, this wasn't a precision issue. It was removing entire elements from the svg that was rendered.

Reply:
That hadn't happened before with our svgs, so maybe something is different in some of our recent icons that we haven't had to deal with. btw, I think turning off this plugin is fine, I'm just curious about what might have been the issue and how best to optimize our svgs going forward. I've found svgs that have high precision in their numeric values often don't actually need that precision, especially at the size of icons. But sometimes they do, so as we add icons we should keep an eye on the final quality and adjust the floatPrecision if needed.

Reply:
Ok, So turning off "convertPathData" doubles the size of the compiled icons.svg file (85kb to 172kb).

@BoulderBrains I ran a test locally of just updating the precision to 3 as @emuvente suggested and that seemed to preserve the new filter-toggle graphic. There may have been some degradation to the "browse-toggle" graphic but the warping wasn't really noticeable the way it was before.

Could you please try to just change the floatPrecision to 3 (still increases the file size to about 100kb)?

Reply:
FYI: @emuvente I've gone through, removed the convertPataData and updated the floatPercision to 3. This method preserved the correct shape of the svgs I was working with.

Reply:
@emuvente in the end upping the floatPrecision got us close enough! as mentioned above, the file size will increase a little bitt but not as much as turning off convertPathData did.